MIGRATION.md

```diff
@@ -147,23 +204,80 @@ There is no codemod for these changes, migrate manually through search and repla
 
-- enforced labels
-- throwing errors + escape hatch
-- as prop in headlines (also mentioned in typography)
+In addition to its increased height, the `Button`'s default size was renamed from `mega` to `giga` to align it with the new size values (see the table above). (🤖 button-default-size)
 
 ### Other changes
 
 - The **NotificationBanner** component has been renamed to **NotificationCard**. (🤖 _component-names-v3_)
-- The **TableRow**, **TableHeader** and **TableCell** components are not exported anymore. Use the **Table** component instead.
-- The **Button**, **IconButton**, (...) and **Hamburger** components' dimensions have been harmonized.
-- The **SelectorGroup**'s `label` is now visible by default, pass `hideLabel` to hide it visually.
-- (...)
+- Label prop names across components were harmonized to follow the _actionLabel_ pattern. (🤖 label-prop-names)
+  - **CardHeader**: `labelCloseButton` 👉 `closeButtonLabel`
+  - **Hamburger**: `labelActive` 👉 `activeLabel`, `labelInActive` 👉 `inactiveLabel`
+  - **Tag**: `labelRemoveButton` 👉 `removeButtonLabel`
+  - **Toggle**: `labelChecked` 👉 `checkedLabel`, `labelUnchecked` 👉 `uncheckedLabel`
+- The **TableRow**, **TableHeader** and **TableCell** components are no longer exported. Use the **Table** component instead.
+- The **Table**'s custom `onSortBy` method signature has been changed. The `nextDirection` argument moved to the third position (`(index, nextDirection, rows)` 👉 `(index, rows, nextDirection)`) and is now optional (i.e. it can be `undefined` instead of `null` in the previous implementation).
+- The **SelectorGroup**'s `label` is now visible by default, pass `hideLabel` to hide it visually. Its children are now also rendered horizontally by default.
+- Default `data-testids` are no longer built into the **Table** and **CardHeader** components. We recommend [querying by role](https://testing-library.com/docs/queries/about/#priority) in tests instead, for them to resemble how users interact with our applications.
+
+### Cleaning up
+
+Finally, Circuit v3 removes previously deprecated and/or unused features and props. These breaking changes may not affect your application if you've already addressed the deprecation warnings in Circuit v2 minors, but we still recommend going through the list of changes below.
+
+- The deprecated **Spacing** component has been removed. Use the **spacing** style mixin instead.
+- The **ProgressBar**'s deprecated `children` prop has been removed. Use the label prop instead.
+- The **Card**'s deprecated `shadow` prop has been removed. Shadows have been replaced by a single outline in an earlier minor version.
+- The deprecated `styleHelpers` aggregate is no longer exported. Import each style mixin directly instead.
+- The `themePropType` is no longer exported from `@sumup/circuit-ui`. Import it from `@sumup/design-tokens` instead.
+- The deprecated **withComponents** HOC has been removed. Use the **useComponents** hook instead.
```

Comment:
```suggestion
- The deprecated `withComponents` HOC has been removed. Use the `useComponents` hook instead.
```

**Edit**: Mhm, I see that you/we've used bold for component names and code for prop names. Not sure what to use here 🤔

Reply:
Same 😄 I was also unsure so I "decided" to make any exports **bold** and props `code`. What do you think?

Reply:
Works for me.